bootstrap/config/kfctl_existing_arrikto.yaml

```diff
@@ -1,105 +1,221 @@
+# This is the config to install Kubeflow on an existing K8s cluster, with support
```

Comment:
Do you want to modify kfctl_existing_arrikto.yaml? My suggestion would probably be to only edit the 0.6.2 files so that you don't risk breaking anyone using the kfctl_existing_arrikto.yaml config from the tip of the branch

Reply:
@jlewi the docs point to the latest tag (v0.6.1 in this case).
As such, I wouldn't expect anything to break in docs.

Excerpt from docs: 
```export CONFIG="https://raw.githubusercontent.com/kubeflow/kubeflow/v0.6.1/bootstrap/config/kfctl_existing_arrikto.0.6.yaml"```

Reply:
I also edited the v0.6.2 file.
I am not sure what the suggestion is.
Won't the config come from a tagged commit (eg v0.6.2)?
If yes, what is the need for having tagged config files?
Better UX?